docs/setup/settings.asciidoc

```diff
@@ -58,6 +58,7 @@ startup. Your Kibana users still need to authenticate with Elasticsearch, which
 `server.ssl.certificateAuthorities`:: List of paths to PEM encoded certificate files that should be trusted.
 `server.ssl.supportedProtocols`:: *Default: TLSv1, TLSv1.1, TLSv1.2*  Supported protocols with versions. Valid protocols: `TLSv1`, `TLSv1.1`, `TLSv1.2`
 `server.ssl.cipherSuites`:: *Default: ECDHE-RSA-AES128-GCM-SHA256, ECDHE-ECDSA-AES128-GCM-SHA256, ECDHE-RSA-AES256-GCM-SHA384, ECDHE-ECDSA-AES256-GCM-SHA384, DHE-RSA-AES128-GCM-SHA256, ECDHE-RSA-AES128-SHA256, DHE-RSA-AES128-SHA256, ECDHE-RSA-AES256-SHA384, DHE-RSA-AES256-SHA384, ECDHE-RSA-AES256-SHA256, DHE-RSA-AES256-SHA256, HIGH,!aNULL, !eNULL, !EXPORT, !DES, !RC4, !MD5, !PSK, !SRP, !CAMELLIA*. Details on the format, and the valid options, are available via the [OpenSSL cipher list format documentation](https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT)
+`server.ssl.redirectHttpFromPort` :: Kibana will start an http server on this port and redirect all requests to the https port. This cannot be the same port as the https port.
```

Comment:
Can you clarify "https port"?  Perhaps just "Kibana will bind to this port and redirect all http requests to https over the port configured as `server.port`"

Can you also move this up in the list to below `keyPassphrase` rather than burying it under things most people don't configure, like cas, protocols, and ciphers.